Cargo.toml

```diff
@@ -21,20 +21,19 @@ name = "e2e-tests"
 path = "e2e_tests/main.rs"
 
 [dependencies]
+tokio = { version = "0.2", features = ["macros"] } # Only used in bin/e2e, not used in library.
```

Comment:
is `bin/e2e` the same as the `binaries` comment on line 26? if so i'd nit them to be consistent

Reply:
Nah, multiple binaries (e.g. both bin/e2e and bin/example) depend on `clap`. But only one binary uses `tokio`.